org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/compare/JavaCompareWithEditionActionImpl.java

```diff
@@ -67,7 +66,7 @@ public void run(ISelection selection) {
 			ci.setHelpContextId(IJavaHelpContextIds.COMPARE_ELEMENT_WITH_HISTORY_DIALOG);
 			CompareUI.openCompareDialog(ci);
 		} else {
-			TeamUI.showHistoryFor(PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage(), input, pageSource);
+			TeamUI.showHistoryFor(JavaPlugin.getActiveWorkbenchWindow().getActivePage(), input, pageSource);
```

Comment:
Shouldn't this be JavaPlugin.getActivePage() directly?

Reply:
Done